src/machine.rs

```diff
@@ -235,7 +236,8 @@ impl<'m> Machine<'m> {
     pub async fn stop(&mut self) -> Result<(), Error> {
         let vm_id = self.vm_id();
         trace!("{vm_id}: Killing VM...");
-        self.child.kill().await?;
+        let child = heim::process::get(self.pid.try_into().unwrap()).await?;
```

Comment:
Let's not `unwrap` the conversion error even if unlikely to happen. Let's just wrap [TryFromIntError](https://doc.rust-lang.org/nightly/std/num/struct.TryFromIntError.html) in our error.

Reply:
done